static/js/components/ExpandedBootcampDisplay.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import _ from "lodash"
+import moment from "moment"
+import striptags from "striptags"
+import { AllHtmlEntities } from "html-entities"
+import ClampLines from "react-clamp-lines"
+
+import { minPrice } from "../lib/courses"
+import { embedlyThumbnail } from "../lib/url"
+import { languageName } from "../lib/util"
+
+import type { Bootcamp } from "../flow/discussionTypes"
+
+const BOOTCAMP_IMAGE_DISPLAY_HEIGHT = 239
+const BOOTCAMP_IMAGE_DISPLAY_WIDTH = 440
+const entities = new AllHtmlEntities()
+
+type Props = {
+  bootcamp: Bootcamp
+}
+
+export default class ExpandedBootcampDisplay extends React.Component<Props> {
```

Comment:
I have the same question here - if it's possible to have an `ExpandedLearningResourceDisplay` instead of two different very similar components I think that would be preferable

Reply:
✔️